account_statement_import_sheet_file/models/account_statement_import_sheet_parser.py

```diff
@@ -62,9 +64,12 @@ def parse(self, data_file, mapping, filename):
         if not lines:
             return currency_code, account_number, [{"transactions": []}]
 
-        lines = list(sorted(lines, key=lambda line: line["timestamp"]))
-        first_line = lines[0]
-        last_line = lines[-1]
+        if lines[0]["timestamp"] > lines[-1]["timestamp"]:
```

Comment:
Why this?

Reply:
I don't really know why, this change was made at https://github.com/OCA/bank-statement-import/pull/559 but I think it is not necessary and can be left as it was.